tests/conftest.py

```diff
@@ -4962,7 +4962,7 @@ def finalizer():
     return driver
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
This looks like quite significant change, do we want to automatically call the `setup_acm_ui` fixture for all executions?

Reply:
Good point